### PR TITLE
billing-services: Add tests for billing-service-bill-fetchBill

### PR DESCRIPTION
**Pull request by krishnendu-eGov**

### CODE REVIEW

File: src/test/java/com/egov/business-services/tests/billing-services-bill.feature [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+     * def noDemandFoundErrorMessage = billingServiceConstants.errorMessages.invalidConsumerBusinessCode.message
+     * def invalidTenantIdErrorCode = billingServiceConstants.errorMessages.invalidTenantId.code
+     * def invalidTenantIdErrorMessage = billingServiceConstants.errorMessages.invalidTenantId.message
+
+
+@fetchbill_01 @fetchbill
+Scenario: Fetch bill with valid consumer code and business service
+    * def fetchBillParams = { consumerCode: '#(consumerCodeBill)', businessService: '#(businessService)', tenantId: '#(tenantId)'}
+    * call read('../preTests/billingServicePretest.feature@customizedParam')
+    * match fetchBillResponse.ResposneInfo.status == '200 OK'
```
Comment (yatheendra-egov) [inline src/test/java/com/egov/business-services/tests/billing-services-bill.feature:34]: '200 ok' is there in commonConstants.yaml under expectedStatus

File: src/test/java/com/egov/business-services/tests/billing-services-bill.feature [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+     * def noConsumerCodeErrorMessage = billingServiceConstants.errorMessages.consumerCode.message
+     * def noBusinessServiceErrorCode = billingServiceConstants.errorMessages.businessService.code
+     * def noTenantIdErrorCode = billingServiceConstants.errorMessages.tenantId.code
+     * def noDemandFoundErrorCode = billingServiceConstants.errorMessages.invalidConsumerBusinessCode.code
+     * def noDemandFoundErrorMessage = billingServiceConstants.errorMessages.invalidConsumerBusinessCode.message
+     * def invalidTenantIdErrorCode = billingServiceConstants.errorMessages.invalidTenantId.code
+     * def invalidTenantIdErrorMessage = billingServiceConstants.errorMessages.invalidTenantId.message
+
+
+@fetchbill_01 @fetchbill
```
Comment (yatheendra-egov) [inline src/test/java/com/egov/business-services/tests/billing-services-bill.feature:30]: please check and add @positive or @negative tags for other test scenarios as well

File: src/test/java/com/egov/business-services/tests/billing-services-bill.feature [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+@fetchbill_WithNoParameters_09 @negative @fetchbill
+Scenario: Fetch bill with no parameters
+    * def fetchBillParams = {}
+    * call read('../preTests/billingServicePretest.feature@customizedParam') 
+    * assert responseStatus == 400
+    * assert fetchBillResponse.Errors[0].code == noBusinessServiceErrorCode
+    * assert fetchBillResponse.Errors[1].code == noTenantIdErrorCode
+   
+@fetchbill_WithInvalidMobileNumber_10 @positive @fetchbill
```
Comment (yatheendra-egov) [inline src/test/java/com/egov/business-services/tests/billing-services-bill.feature:98]: it should be @negative if i'm not wrong

Comment (krishnendu-eGov) [reply src/test/java/com/egov/business-services/tests/billing-services-bill.feature]: Comment addressed

File: src/test/java/com/egov/business-services/tests/billing-services-bill.feature (hunk 2 above)
Comment (krishnendu-eGov) [reply src/test/java/com/egov/business-services/tests/billing-services-bill.feature]: Comment addressed

File: src/test/java/com/egov/business-services/tests/billing-services-bill.feature (hunk 1 above)
Comment (krishnendu-eGov) [reply src/test/java/com/egov/business-services/tests/billing-services-bill.feature]: Comment addressed

**Comment by yatheendra-egov** [comment]
My review comments are addressed

**Pull request merged**